app/src/main/java/app/revanced/integrations/youtube/patches/misc/SpoofStreamingDataPatch.java

```diff
@@ -180,6 +181,15 @@ public static String appendSpoofedClient(String videoFormat) {
         return videoFormat;
     }
 
+    /**
+     * Injection point.
+     */
+    public static void syncLivestreamTime(TextView view) {
+        Utils.runOnMainThreadDelayed(() -> {
+            if (view != null) view.performClick();
```

Comment:
This is not ideal

Because it doesn't work as intended in the following cases:

1. Shorts livestream (in Shorts, the `Skip to Livestream` button is a litho component)
2. When the livestream is playing from a feed, not player (`TimeBarLiveLabel` is not initialized)
3. When the video already starts at the current time (some livestreams start at the current time, not at 0 seconds on iOS clients, so clicking on `TimeBarLiveLabel`, it skips to 0 seconds)